### PR TITLE
Us110934/remove third level nav

### PR DESCRIPTION
**Pull request by TomMichielsD2L**

### CODE REVIEW

File: components/d2l-inner-module.js [hunk 1]
```diff
@@ -198,6 +203,9 @@ class D2LInnerModule extends ASVFocusWithinMixin(PolymerASVLaunchMixin(Completio
 	}
 
 	_onHeaderClicked() {
+		if (this.entity && this.entity.hasClass('hide-description')) {
```
Comment (benyamin-noori) [inline components/d2l-inner-module.js]: You have this part in two places: `this.entity && this.entity.hasClass('hide-description')`. 

Would it be better to compute a property based on this logic and use that property here? 

Or make it into a function and use that?

File: components/d2l-inner-module.js [hunk 2]
```diff
@@ -223,5 +231,13 @@ class D2LInnerModule extends ASVFocusWithinMixin(PolymerASVLaunchMixin(Completio
 			return '';
 		}
 	}
+
+	hideDescription(entity) {
```
Comment (benyamin-noori) [inline components/d2l-inner-module.js]: Nit: Better name for this function?

e.g. "getHideDescriptionClass", "hideDescriptionClass"?

Comment (TomMichielsD2L) [reply components/d2l-inner-module.js]: Done!

File: components/d2l-inner-module.js (hunk 1 above)
Comment (TomMichielsD2L) [reply components/d2l-inner-module.js]: Done!

File: components/d2l-inner-module.js [hunk 3]
```diff
@@ -180,12 +189,17 @@ class D2LInnerModule extends ASVFocusWithinMixin(PolymerASVLaunchMixin(Completio
...
 	}
+
 	getSubEntities(entity) {
 		return entity && entity.getSubEntities()
 			.filter(subEntity => (subEntity.hasClass('sequenced-activity') && subEntity.hasClass('available')) || (subEntity.href && subEntity.hasClass('sequence-description')))
 			.map(this._getHref);
 	}
 
+	_getHideDesciption(entity) {
+		return entity && entity.hasClass('hide-description');
```
Comment (benyamin-noori) [inline components/d2l-inner-module.js]:
```suggestion
		return entity && entity.hasClass('hide-description') || false;
```

**Pull request merged**